torchchat/generate.py

```diff
@@ -788,7 +784,6 @@ def generate(
                 input_pos = input_pos + num_added
                 next_token = next_tokens[-1]
         else:
-            generated_tokens = []
```

Comment:
Not used. Generated tokens are appended after the call to `decode_n_tokens`, but then nothing happens.